src/utils/entraGroup.ts

```diff
@@ -27,19 +43,36 @@ export const entraGroup = {
   /**
    * Get a list of groups by display name.
    * @param displayName Group display name.
+   * @param properties Properties to include in the response.
    */
-  async getGroupsByDisplayName(displayName: string): Promise<Group[]> {
-    return odata.getAllItems<Group>(`${graphResource}/v1.0/groups?$filter=displayName eq '${formatting.encodeQueryParameter(displayName)}'`);
+  async getGroupsByDisplayName(displayName: string, properties?: string): Promise<Group[]> {
+    const queryParameters: string[] = [];
+
+    if (properties) {
+      const allProperties = properties.split(',');
+      const selectProperties = allProperties.filter(prop => !prop.includes('/'));
+
+      if (selectProperties.length > 0) {
+        queryParameters.push(`$select=${selectProperties}`);
+      }
+    }
+
+    const queryString = queryParameters.length > 0
+      ? `?${queryParameters.join('&')}`
```

Comment:
actually we already have the `$filter` query param in the endpoint call so we need to use `&` instead of `?` 

```suggestion
      ? `&${queryParameters.join('&')}`
```

Otherwise we will get an error 
![image](https://github.com/user-attachments/assets/c2beb902-467c-42ac-b6dd-01b1527e0ff3)